_raml/apps/carts/schema/put_carts_cartid_paymentdata_request.schema.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Put Carts CartId PaymentData Request Schema",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "The name of the payment method.<br>Required: `true`"
+    },
+    "priceAmount": {
+      "type": "number",
+      "description": "The costs for the payment method.<br>Required: `false`"
```

Comment:
It can be a cost (positive value) or a discount (negative value).
This behavior fits with real epages payment methods where merchants can specify a cost or a discount for their usage.
note: on the other hand, shipping methods usage can only generate a cost.